src/EnergyPlus/ElectricBaseboardRadiator.cc

```diff
@@ -610,8 +608,7 @@ namespace ElectricBaseboardRadiator {
             MyOneTimeFlag = false;
         }
 
-        if (ElecBaseboard(BaseboardNum).ZonePtr <= 0)
-            ElecBaseboard(BaseboardNum).ZonePtr = state.dataZoneEquip->ZoneEquipConfig(ControlledZoneNumSub).ActualZoneNum;
+        if (ElecBaseboard(BaseboardNum).ZonePtr <= 0) ElecBaseboard(BaseboardNum).ZonePtr = ControlledZoneNum;
```

Comment:
This could move up 2 lines? I am thinking out loud, not suggesting more changes to this branch.